crossdock/main.go

```diff
@@ -21,11 +21,38 @@
 package main
 
 import (
-	"github.com/yarpc/yarpc-go/crossdock/client"
+	"github.com/yarpc/yarpc-go/crossdock-go/crossdock"
+	b "github.com/yarpc/yarpc-go/crossdock/client/behavior"
+	"github.com/yarpc/yarpc-go/crossdock/client/echo"
+	"github.com/yarpc/yarpc-go/crossdock/client/errors"
+	"github.com/yarpc/yarpc-go/crossdock/client/headers"
+	"github.com/yarpc/yarpc-go/crossdock/client/tchclient"
+	"github.com/yarpc/yarpc-go/crossdock/client/tchserver"
 	"github.com/yarpc/yarpc-go/crossdock/server"
 )
 
 func main() {
 	server.Start()
-	client.Start()
+	crossdock.Start(dispatch)
+}
+
+func dispatch(s b.Sink, behavior string, ps b.Params) {
```

Comment:
Instead of renaming the imported package from `behavior` to `b`, just use the name `b` for this variable? Go encourages short variable names where obvious.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names